docs/eggs/configuration.md

```diff
@@ -64,19 +69,31 @@ ignore:
 checkFormat: deno fmt
 checkTests: deno test
 checkInstallation: true
-checkAll: false
+check: false
 ```
 
 ### Field information
 
 - name:
     - The name of your module.
+- entry:
+    - The "index file" of your project. This is what users will see when they try to import your module from our registry!
+    - Defaults to `./mod.ts`
 - description:
     - A description of your module that will appear on the gallery.
+- homepage:
+    - A link to your homepage. Usually a repository.
+    - Defaults to `null`.
+- unstable:
+    - Is this version unstable?
+    - Default value is determined by Semantic Versioning rules.
+- unlisted:
+    - Should people be able to find this module/version on the gallery?
```

Comment:
```suggestion
    - Should people be able to find this module/version on the gallery? If `true`, module is hidden.
```
Just wanted to add this, though implied, as we've gotten some confusion in the past.